### PR TITLE
Translations update

### PR DESCRIPTION
**Pull request by krzychu124**
Another update of translations from Crowdin:

- __Chinese Traditional__ @jrthsr700tmax
- __Portuguese__  BlackScout (BS_BlackScout)
- __Chinese Simplified__ 田七不甜 (TianQiBuTian)
- __Ukrainian, Russian__ - Dmytro Lytovchenko (kvakvs)
- __Turkish__ Tayfun Bilgi (Tayfun [Typhoon])

### CODE REVIEW

File: TLM/TLM/Resources/Translations/Menu.csv [hunk 1]
```diff
@@ -17,14 +17,14 @@
...
+"Tooltip.Keybinds:Auto TL","","Ctrl+Click: Quick setup","","Feux automatiques","","","","","","Ctrl + Klik: Szybka instalacja","Ctrl+Clique: Configuração rápida","Автоматическое создание светофора с таймером","自動化號誌","Ctrl+点击: 快速设置","Ctrl+Click: Quick setup","Özdevinimli TI","Автоматичне створення світлофора з таймером"
+"Tooltip.Keybinds:Add priority signs","","Ctrl+Shift+Click: Quick setup","","Ajouter les panneaux de priorité","","","","","","Ctrl+Shift+Klik: Szybka instalacja","Ctrl+Shift+Clique: Configuração rápida","Добавить знаки приоритета на перекрёстках круга","加入優先標誌","Ctrl+Shift+点击: 快速设置","Ctrl+Shift+Click: Quick setup","Öncelik işaretleri ekle","Додати знаки приорітету руху"
```
Comment (kvakvs) [inline TLM/TLM/Resources/Translations/Menu.csv]: A problem right here: My own translations do not include shortcut keys.
We can either modify Rus and Ukr and include shortcuts, or modify the code (simple but incorrect) and make it build localized shortcut strings (correct but harder).

Comment (krzychu124) [reply TLM/TLM/Resources/Translations/Menu.csv]: Yeah, I saw that, mentioned about it before, but now I don't know what to do.

Comment (kvakvs) [reply TLM/TLM/Resources/Translations/Menu.csv]: Updated 3 strings above

**Comment by krzychu124** [comment]
I've just tested, looks good

**Pull request merged**